### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by mesudpasic**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  initializeListener()
+  res.status(200).send({ message: 'Websocket listener initialized' })
+})
+
+// handle exceptions
+app.use(async (err, req, res, next) => {
+  if (res.headersSent) {
+    return next(err)
+  }
+  const errCode = err.status || 401
```
Comment (europaul) [inline src/app.js:43]: why not test equality?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+[Makefile]
+indent_style = tab
+indent_size = 4
+
+[*.xml]
+indent_size = 4
+
+[*.yml,*.yaml]
+indent_size = 4
```
Comment (europaul) [inline .editorconfig:22]: please end all files with a newline

File: src/utils/env.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const env = (key, defaultValue, isBoll = false) => {
```
Comment (europaul) [inline src/utils/env.js]: did you mean `isBool`?

**Review by europaul — CHANGES_REQUESTED** [review]

File: src/app.js (hunk 1 above)
Comment (mesudpasic) [reply src/app.js:43]: this is assignment. no need to change it.

File: .editorconfig (hunk 2 above)
Comment (mesudpasic) [reply .editorconfig:22]: format command is run on all scripts, I can't manually add new lines int he code.

File: src/utils/env.js (hunk 3 above)
Comment (mesudpasic) [reply src/utils/env.js]: yeah, that was inherited from boilerplate v1.0

File: src/app.js (hunk 1 above)
Comment (europaul) [reply src/app.js:43]: I meant `err.status === 401`

Comment (mesudpasic) [reply src/app.js:43]: please leave it as is, I'm sending the code in err.status if for some reason is not present 401 is taken.

File: .editorconfig (hunk 2 above)
Comment (europaul) [reply .editorconfig:22]: as far as I see from `.editorconfig:9` every file should end with a newline

File: src/app.js (hunk 1 above)
Comment (europaul) [reply src/app.js:43]: oh, that's what it means...

File: .editorconfig (hunk 2 above)
Comment (mesudpasic) [reply .editorconfig:22]: like I said I run npm run format on every module and pushed it as such.,

Comment (europaul) [reply .editorconfig:22]: please adjust all the files to the format defined in `.editorconfig`

Comment (mesudpasic) [reply .editorconfig:22]: what do you want me to do here, I run npm run format and lint not sure what else I should be doing here.

Comment (mesudpasic) [reply .editorconfig:22]: new line is there now

**Review by europaul — APPROVED** [review]

**Pull request merged**